### PR TITLE
Quilljs Rich Text Editor Trace Improved

### PR DESCRIPTION
**Pull request by dhavalveera**
**Examples of `link[href]`**:

https://my.iccsafe.org/login
https://games.steelseries.com/home

**Example of `div` class**:

https://zenoamaro.github.io/react-quill/

### CODE REVIEW

File: src/technologies/q.json [hunk 1]
```diff
@@ -350,6 +350,7 @@
     "js": {
       "Quill": ""
     },
+    "dom": "link[href*='cdn\\.quilljs.com/'], div.quill, div.ql-editor",
```
Comment (nurbek91) [inline src/technologies/q.json:353]: lets replace to this one
```
"dom": {
      "link[href*='cdn.quilljs.com/']": {
        "attributes": {
          "href": "cdn\\.quilljs\\.com/([\\d\\.]+)/\\;version:\\1"
        }
      }
    },
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate , thanks!!

File: src/technologies/q.json (hunk 1 above)
Comment (dhavalveera) [reply src/technologies/q.json:353]: what about `div.quill` & `div.ql-editor` => this trace is required where CDN isn't being used

**Review by nurbek91 — APPROVED** [review]

File: src/technologies/q.json [hunk 2]
```diff
@@ -346,6 +346,23 @@
       24
     ],
     "description": "Quill is a free open-source WYSIWYG editor.",
+    "dom": {
+      "div[class*='ql-editor']": {
+        "attributes": {
```
Comment (AliasIO) [inline src/technologies/q.json:351]: This block can be changed to `"exists": ""`.

File: src/technologies/q.json [hunk 3]
```diff
@@ -346,6 +346,23 @@
...
+          "class": "ql-editor"
+        }
+      },
+      "div[class*='quill']": {
+        "attributes": {
```
Comment (AliasIO) [inline src/technologies/q.json:356]: This block can be changed to `"exists": ""`.

File: src/technologies/q.json (hunk 2 above)
Comment (dhavalveera) [reply src/technologies/q.json:351]: replace the `attributes` to `exists`?

File: src/technologies/q.json (hunk 3 above)
Comment (dhavalveera) [reply src/technologies/q.json:356]: replace the `attributes` to `exists`?

Comment (nurbek91) [reply src/technologies/q.json:356]:
```
"div[class*='ql-editor']": {
        "attributes": { 
             "class": "quill"
        }
      },
"div[class*='quill']": {
        "attributes": { 
             "class": "quill"
        }
      },
```
something similar
```
"div[class*='quill'], div[class*='ql-editor'] ": {
"exists": ""
}
```

Comment (dhavalveera) [reply src/technologies/q.json]: @AliasIO, @nurbek91  - please review this updated PR, is this correct what you mentioned above

Comment (nurbek91) [reply src/technologies/q.json]: @dhavalveera  Does the code work locally on the examples?

Comment (dhavalveera) [reply src/technologies/q.json]: yes, it's working....

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**